ptvsd/wrapper.py

```diff
@@ -1252,30 +1279,36 @@ def on_setBreakpoints(self, request, args):
         self.path_casing.track_file_path_case(path)
         src_bps = args.get('breakpoints', [])
 
+        bp_type = 'python-line'
+        if not path.lower().endswith('.py'):
+            if self.launch_options.get('DJANGO_DEBUG', False):
```

Comment:
This doesn't apply to launch debugging only, attaching too would need this check, hence the suggestion to change property name